docker/README.md

```diff
@@ -1,47 +0,0 @@
-# Docker setup for OpenFermion and select plugins
```

Comment:
I'm not sure why you deleted these instructions. Why don't you just add the part that is new and keep these old instructions otherwise. I spent some time polishing these instructions last time. Not to mention the issues I mention below, e.g. calling the package "FermiLib".

Reply:
I will add those back - sorry about the typos and hasty deletes

Reply:
No problem. Did you see other comments? I don't mean to rush you but since you addressed these but not the others I wanted to check.

Reply:
Yes I saw all the comments. I will also get Psi4 and PySCF to work in the new dockerfile. The thing with capital D in Dockerfile is just that Docker recognizes "Dockerfile" as a default file name. Otherwise the build command will need to specify the name of the file for building the Docker image. It's not a big deal though.

Reply:
ok yeah sorry. The "Dockerfile" -> "dockerfile" comment was the one I cared about least ;-)